sources/packages/api/src/route-controllers/institution/models/institution.dto.ts

```diff
@@ -0,0 +1,85 @@
+import { IsNotEmpty, IsOptional } from "class-validator";
+
+export class CreateInstitutionDto {
+  @IsNotEmpty()
+  legalOperatingName: string;
```

Comment:
The read-only data that is retrieved from BCeID Web Service should not be expected to be received from the API client.
User first name, last name, institution business GUID, and institution operating name should be retrieved during the institution creation.

Reply:
DIdnt get this , lets discuss

Reply:
The data that is received from BCeID Web Service and is displayed to the client in the UI, should again be retrieved during the user creation in the API. We should only allow the client to update the data that he is able to edit.
The source of truth for the read-only data is BCeID Web Service and in my option, the data is going to be persisted to our DB should come from BCeID Web Service and not from the UI.

Reply:
Done